webapp/store/snap_details_views.py

```diff
@@ -17,6 +19,29 @@
 )
 from webapp.markdown import parse_markdown_description
 
+yaml = YAML(typ="safe")
+
+
+def _get_file(file):
```

Comment:
@jpmartinspt I feel that this should be moved in the helper file since this function exists else where